### PR TITLE
Delete all build files to start from the scratch with a more controlled build system

### PR DESCRIPTION
**Pull request by felixortegam**
Delete all build files to start from the scratch with a more controlled build system

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,48 +1,43 @@
...
-  test: require(path.join(__dirname, 'cfg/test'))
+module.exports = {
+    entry: [
+      'webpack/hot/dev-server',
+      'webpack-dev-server/client?http://localhost:8080',
+      path.resolve(__dirname, 'src/components/app/app.js')
+    ],
+    resolve: {
+      extensions: ['', '.js', '.jsx']
+    },
```
Comment (javierlopezdeancos) [inline webpack.config.js:13]: we can't resolve image or scss, svg, ttf files or some static file types with web pack?

File: webpack.config.js [hunk 2]
```diff
@@ -1,48 +1,43 @@
...
+    output: {
+        path: path.resolve(__dirname, 'build'),
+        filename: 'bundle.js',
+    },
```
Comment (javierlopezdeancos) [inline webpack.config.js:17]: we don't should have the name for final js file like...katanga.js or something like that
?

File: webpack.config.js (hunk 1 above)
Comment (felixortegam) [inline webpack.config.js:13]: Sorry, i´m still looking a solution for this.

File: webpack.config.js (hunk 2 above)
Comment (felixortegam) [inline webpack.config.js:17]: Sure, we should change the name of bundle file to katanga.js. We´ll do it.

**Pull request merged**